README.md

```diff
@@ -70,7 +71,49 @@ Before beginning a sprint:
 
 During a sprint we track progress using a [scrum or kanban-like board](#notes)<sup>4</sup> with status columns for To Do, In Progress, Blocked, Ready for Review, Ready for QA, and Done. 
 
-After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs. 
+After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
+
+## Velocity Tracking
+
+### What is Velocity Tracking?
+
+Velocity tracking is the practice of using our average historical output to inform how we plan and commit to upcoming work. 
+
+Velocity is a number that represents the amount of work completed by a team during a specific time period. Typically, we talk about “sprint velocity” and “average sprint velocity”. `sprint-velocity` describes the amount of work completed in a single sprint. `average-sprint-velocity` is an average of the previous *n* sprints (where *n* is something like 2 - 4 sprints). We use the latter during sprint planning to determine how much work to commit to in the sprint being planned.
+
+### How Do We Track Velocity?
+
+Velocity tracking stuff mostly occurs during sprint planning. 
+
+If the sprint you're planning is the first sprint where you'll be incorporating anything related to velocity you need to choose an initial "fake" `average-sprint-velocity`. Its value might be something like 0.5 (avg. number of points completed per person per hour).
+
```

Comment:
For the very first sprint that you start tracking velocity, you need to choose a target "seed velocity" to which your team commits. For example, your seed velocity might be 0.5 (this is the target avg. number of points completed per person per hour of a sprint). Over time, this will be replaced by the *actual* average sprint velocities accomplished in prior sprints.